.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
Is this necessary? I don't see it in cosmic-comp for example. Is it gonna cause any warnings if nix is not installed but direnv is?

Reply:
I guess it's not necessary, but my workflow depends on it. I use an extension to vscode that loads `.envrc`, and i can't easily work without it (vscode doesn't really launch "inline" with the terminal environment, so launching from `nix develop` does not do the trick as it would with i.e. nvim). It could just as well be gitignored, though. I don't see it causing an issue without `nix-direnv`, as i tried just now making a dummy directory with and `.envrc` of `use AAAAAAAAAAAAAAAAA` and it doesn't complain at all. firstly; it asks before even trying to load anyway, but when it does load, it seems to just succeed with no additional notes.

Reply:
I guess you can add .direnv and .envrc to your global gitignore, then it doesn't have to be added to individual projects.

Reply:
hm. that's... fair enough. i'll admit, i do not have extensive knowledge of git, and was genuinely unaware of this. always thought people just manually made sure to keep editor metadata and such untracked or something. should i remove that commit from my branch then, and force push, or is there some "cleaner"(?) way to do it?